tests/integration/ssh/test_raw.py

```diff
@@ -10,16 +10,19 @@
 # Import Salt Libs
 import salt.utils.platform
 
+import timeout_decorator
+
 
 @skipIf(salt.utils.platform.is_windows(), 'salt-ssh not available on Windows')
 class SSHRawTest(SSHCase):
     '''
     testing salt-ssh with raw calls
     '''
+    @timeout_decorator.timeout(60, use_signals=False)
     def test_ssh_raw(self):
         '''
         test salt-ssh with -r argument
         '''
-        msg = 'running raw msg'
+        msg = 'password: foo'
```

Comment:
can we ensure we are testing for both of these msg types?

Also can you elaborate why you need this decorator?

Reply:
Is there any need to test for the text `running raw msg`? It seems to just be some text to test for, where as this PR puts it to use as also triggering the #25721 bug.